clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -1237,21 +1241,26 @@ describe("Fides-js TCF", () => {
     });
   });
 
-  describe("User preference sources of truth for UI and CMP API", () => {
+  /**
+   * There are the following potential sources of user preferences:
+   * 1) fides_string override option (via config.options.fidesString)
+   * 2) DEFER: preferences API (via a custom function)
+   * 3) local cookie (via fides_consent cookie)
+   * 4) "prefetched" experience (via config.options.experience)
+   * 5) experience API (via GET /privacy-experience)
+   *
+   * These specs test various combinations of those sources of truth and ensure
+   * that Fides loads the correct preferences in each case.
+   */
+  describe("user preferences overrides", () => {
```

Comment:
The specs below are really dense and hard to understand; I added some comment blocks to check my understanding as I reworded them a bit,.